Code/Framework/AzFramework/AzFramework/Physics/Configuration/JointConfiguration.h

```diff
@@ -50,6 +52,10 @@ namespace AzPhysics
         AZ::Crc32 GetChildLocalPositionVisibility() const;
         AZ::Crc32 GetStartSimulationEnabledVisibility() const;
 
+        // Allow additional values specific to a particular physics backend to be retrieved and modified.
+        virtual AZStd::optional<float> GetPropertyValue([[maybe_unused]] const AZ::Name& propertyName) { return AZStd::nullopt; }
+        virtual void SetPropertyValue([[maybe_unused]] const AZ::Name& propertyName, [[maybe_unused]] float value) {};
```

Comment:
nit: move trailing `;` (also suggest even though it's a bit of a pain, it likely makes sense to put the braces on a new line)